src/visualize_correlation.py

```diff
@@ -0,0 +1,31 @@
+import matplotlib.pyplot as plt
+import seaborn as sns
+import pandas as pd
+import click
+
+
+@click.command()
+@click.option('--file_path')
```

Comment:
maybe add some information at here so people know wat is your input. Here is how tiff do it: https://github.com/ttimbers/breast_cancer_predictor_py/blob/v2.0.0/scripts/evaluate_breast_cancer_predictor.py